src/electron/main/native-high-contrast-mode-listener.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { UserConfigMessageCreator } from 'common/message-creators/user-config-message-creator';
+import { NativeTheme } from 'electron';
+
+export class NativeHighContrastModeListener {
+    private lastHighContrastUpdate: boolean;
```

Comment:
the combination of name + type seems weird to me. From the name (specially the **update**, I'd expected this to hold a timestamp (maybe); from the type, I'd expect this to hold a flag to signal if we already process the update (like sending a message so the whole UI change theme)

Reply:
Updated to `highContrastStateAtLastNotify`